### DISCUSSION TITLE
How does one go about activating the dns exploit?

### DISCUSSION THREAD

**Discussion by thebossjeff** [3kh0/ext-remover#648] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
i change the servers, get to the gstatic page, and am prompted with a black page stating that the "incognito exploit" will not function in its entirety on my device. Very interesting indeed.

**Comment by S-PScripts** [comment]
what is this DNS exploit? vague

if its incognito im not sure how you got that
